File: solutions/go/isogram/1/isogram.go
package isogram

import "strings"

func IsIsogram(word string) bool {
	arrWord := []rune(strings.ToLower(word))

	for i := 0; i < len(arrWord)-1; i++ {
		if arrWord[i] == '-' || arrWord[i] == ' ' {
			continue
		}
		for ii := i + 1; ii < len(arrWord); ii++ {
			if arrWord[i] == arrWord[ii] {
				return false
			}
		}
	}
	return true
}

File: solutions/c/isogram/1/src/isogram.c
#include "isogram.h"
#include "string.h"
#include "ctype.h"


bool is_isogram(const char phrase[])
{
    int length;
    if(phrase==NULL)return false;

    for(length=0;phrase[length]!='\0';length++);
    if (length==0)return true;

    int i;
    for(i=0;i<length;i++){
        int ii;
        if( isalpha(phrase[i])==false)continue;
        for(ii=i+1;ii<length;ii++)
        {
            if(tolower(phrase[i])==tolower(phrase[ii])) return false;
        }
    }

    return true;
}